Makefile

```diff
@@ -23,7 +23,22 @@ fmt:
 	goimports -w -l -local github.com/autokitteh/starlark-lsp cmd/ pkg/
 
 install:
-	go install ./cmd/starlark-lsp
+	go install ./cmd/autokitteh-starlark-lsp
 
 builtins:
 	go run ./hack/starlark-builtins.go > pkg/analysis/builtins.py
+
+
+.PHONY: sysroot-pack sysroot-unpack release-dry-run release
+
+PACKAGE_NAME          := github.com/autokitteh/autokitteh-starlark-lsp
+GOLANG_CROSS_VERSION  ?= v1.21  # could use latest, but it's better to specify exact Go version
+
+DOCKER_RUN = docker run --rm -e CGO_ENABLED=1 -v /var/run/docker.sock:/var/run/docker.sock -v `pwd`:/go/src/$(PACKAGE_NAME) -v `pwd`/sysroot:/sysroot -w /go/src/$(PACKAGE_NAME)
```

Comment:
This won't work (mounting the docker socket) if we move to our hosted gitlab runners (which we already considered before) unless we will allow privileged access on the runners (which we won't)

so fine for now as long as GitHub allows it, but keep a mental note we might need to change in the future

Reply:
I think there will be no problems once we decide to change it.
docker should be able to run and publish to github.

Reply:
the problem is with mounting the docker socket and talking to the host
it requires root privileges on the host 

but for now, if it works, fine by me

Reply:
it works for now, yes. But if you have a better solution, please suggest. The only assumption now that we need a docker with cross-compilers installed.